### ISSUE TITLE
Firefox extension fails to load chrome-gnome-shell with non /usr install

### ISSUE THREAD

**Issue by prahal** [nE0sIghT/chrome-gnome-shell-mirror#47]
tl;dr
I fixed this issue by moving
`/usr/local/lib/mozilla/native-messaging-hosts/org.gnome.chrome_gnome_shell.json`
to
`/usr/lib/mozilla/native-messaging-hosts/org.gnome.chrome_gnome_shell.json`

With chrome-gnome-shell extension installed from git and cmake prefix to /usr/local per instructions I also get:

> "Attempt to postMessage on disconnected port"

this is a dup of:
[Firefox addons site review 847581](https://addons.mozilla.org/En-us/firefox/addon/gnome-shell-integration/reviews/847581/)
chrome-gnome-shell is not called at all and Firefox report it cannot start org.gnome.chrome_gnome_shell as not allowed or not found.

From https://wiki.mozilla.org/WebExtensions/Native_Messaging and https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Native_messaging I nailed it down to:
/usr/local/lib/mozilla/native-messaging-hosts/org.gnome.chrome_gnome_shell.json
not a valid location for the manifest to allow Firefox to start chrome-gnome-shell .
Allowed location are:
- for global visibility:
```
/usr/lib/mozilla/native-messaging-hosts/<name>.json
/usr/lib64/mozilla/native-messaging-hosts/<name>.json

```
- for per-user visibility:
`~/.mozilla/native-messaging-hosts/<name>.json`

**Comment by nE0sIghT** [comment]
Fixed in wiki. Thanks
